demo/App.tsx

```diff
@@ -140,13 +154,14 @@ const App = () => {
                 color="green"
                 testID="replaceCustomDataBtn"
                 accessibilityLabel="replaceCustomDataBtn"
-                onPress={() =>
-                  RaygunClient.updateCustomData(data => {
-                    console.log('Existing customData', data);
-                    return {
-                      replaceAllData: true
-                    };
-                  })
+                onPress={() => {
+                  const updater = (data: CustomData) => {
+                    // Do something with the custom data if you wish
+                    console.log("DATA:", data);
+                    return data
```

Comment:
I think a real-use case would be good to include here

Reply:
Adding in some functionality that remove numbers and arrays

DONE